onnxruntime/contrib_ops/cuda/bert/flash_attention/static_switch.h

```diff
@@ -23,6 +23,36 @@
     }                                           \
   }()
 
+#ifdef FLASHATTENTION_DISABLE_ALIBI
```

Comment:
Shall we define FLASHATTENTION_DISABLE_ALIBI to disable ALIBI since it is not used yet?